source/reference/modules/deployer/index.rst

```diff
@@ -355,6 +355,52 @@ the steps described in :ref:`debugging-deployer-issues`
   Changing or deleting a processed commit file could cause unchanged files to be indexed again and
   it should be done as a last resort in case of errors.
 
+.. _jacket:
+
+^^^^^^
+Jacket
+^^^^^^
+Indexing rich document content into a single search entry greatly improves searchability in your project.
+Crafter Deployer is able to index the content of a rich document (e.g. PDF, DOC, DOCX, PTT, etc.) along with metadata
```

Comment:
@alhambrav it might be good to try to explain something like this, since there's still some confusion on where the jacket data ends up when indexed: `jackets are basically only additional metadata for content. When a binary file (which includes documents like pdfs and word files) is indexed, the XML of the jacket, along with the content of the binary file, are indexed under the path of the binary file. E.g. when indexing file `/static-assets/documents/contracts/2024-contract.pdf`, the Deployer resolves its jacket at `/site/documents/contracts/2024-contract.xml`, extracts the XML conent of the jacket, and indexes everything under `/static-assets/documents/contracts/2024-contract.pf`